lib/game_teams.rb

```diff
@@ -27,5 +28,6 @@ def initialize(row)
     @shots = @shots.to_i
     @tackles = @tackles.to_i
     @takeaways = @takeaways.to_i
+    @team_id = @team_id.to_i
```

Comment:
We might run into issues with the data type for team_id and maybe other places.  I used team_id as a string.  We can fix that easily though.